test/test_helper.rb

```diff
@@ -1,4 +1,3 @@
-$LOAD_PATH.unshift File.expand_path("../lib", __dir__)
```

Comment:
@seuros I remember that by removing this line, gem files are loaded from installed version, instead of current directory. Any other reason to remove it ?

Reply:
`bundle exec rake `